### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by MacMarMor**
209399

### CODE REVIEW

**Comment by MacMarMor** [comment]
Lab2

File: src/main/java/edu/kis/powp/jobs2d/drivers/command/ComplexCommand.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.drivers.command;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand{
+    ArrayList<DriverCommand> cmd;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/command/ComplexCommand.java:7]: Should be private and type should point to the interface (i.e., List)

File: src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListenerThree.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public void actionPerformed(ActionEvent e) {
+        Job2dDriver currentDriver = driverManager.getCurrentDriver();
+
+        ArrayList<DriverCommand> driverCommands = new ArrayList<>(Arrays.asList(
+                new SetPositionCommand(currentDriver, -100, -100),
+                new OperateToCommand(currentDriver, -75, 100),
+                new OperateToCommand(currentDriver, 125, 100),
+                new OperateToCommand(currentDriver, 100, -100),
+                new OperateToCommand(currentDriver, -100, -100))
+        );
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListenerThree.java:34]: This is not a factory (3.1.5 is missing)

**Review by kowallus — COMMENTED** [review]
3.5 pts

Specific remarks are inlined in the code.

**Pull request closed without merging**